.github/workflows/ci.yml

```diff
@@ -27,6 +27,34 @@ jobs:
     outputs:
       check_if_build: ${{ steps.check.outputs.check_if_build }}
 
+  build_lambda:
+    name: Build (Lambda)
+    if: (github.event_name == 'pull_request') || (github.event_name == 'push' && github.ref == 'refs/heads/main')
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./lambda
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Install cargo-lambda
+        run: pip install cargo-lambda
```

Comment:
why not from crates.io? https://crates.io/crates/cargo-lambda

Reply:
From [cargo-lambda.info](https://www.cargo-lambda.info/guide/installation.html)
> WARNING
cargo-install compiles the binary in your system, which usually takes more than 10 minutes. This method doesn't install [Zig](https://ziglang.org/) either, which is a requirement if you want to cross compile packages from macOS or Windows to Lambda Linux environments.

Reply:
ok...

Reply:
It's not ideal. I'll look into alternatives.

Reply:
let's not spend much time on this. If it works, it's ok and it actually simpler then installing zig or anything.